best-practices-cel/disallow-default-namespace/.chainsaw-test/chainsaw-test.yaml

```diff
@@ -0,0 +1,55 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/kyverno/chainsaw/main/.schemas/json/test-chainsaw-v1alpha1.json
+apiVersion: chainsaw.kyverno.io/v1alpha1
+kind: Test
+metadata:
+  creationTimestamp: null
+  name: disallow-default-namespace
+spec:
+  steps:
+  - name: step-01
+    try:
+    - apply:
+        file: ../disallow-default-namespace.yaml
+    - patch:
+        resource:
+          apiVersion: kyverno.io/v1
+          kind: ClusterPolicy
+          metadata:
+            name: disallow-default-namespace
+          spec:
+            validationFailureAction: Enforce
```

Comment:
I wonder if it is possible to write the policies in the `Enforce` mode so that we don't patch it while testing within chainsaw. 
cc @chipzoller

Reply:
No, we want all validate rules (or the ones which make sense) to be in `Audit` mode for the library. It's fine to use a patch in Chainsaw, we do it everywhere now.